src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -224,37 +228,39 @@ def _create_users_tasks(self) -> List[asyncio.Task[None]]:
             )
         return tasks
 
-    def _get_upper_bound(self, batteries: Set[int]) -> float:
+    def _get_upper_bound(self, batteries: Set[int], use_all: bool) -> float:
         """Get total upper bound of power to be set for given batteries.
 
         Note, output of that function doesn't guarantee that this bound will be
         the same when the request is processed.
 
         Args:
             batteries: List of batteries
+            use_all: whether all batteries in the power request must be used.
```

Comment:
I am wondering if `use_all` is the best name for this. Technically it's correct, but the underlying impact of that is stronger than the name implies, no? 

I wouldn't understand immediately, that `use_all` here implies that it refers to specifically problematic batteries.
Additionally, it could also mean to use all available batteries at the location, not just the ones in the `batteries` set 

how about "ignore_invalid_batteries` or something along those lines?

Reply:
true true,  I'll update it to `must_use_all` if that sounds good to you both

Reply:
Sorry for bikeshedding, but I think a very common term is ˋforceˋ, so what about ˋforce_use_allˋ? Or maybe ˋinclude_brokenˋ to make it even more strongly about using broken stuff. I'm fine to keep ˋmust_use_allˋ too, just a mild suggestion.

Reply:
I'll go with `force_use_all`

Reply:
Woops! Now after a review on all the code I do think `include_broken` is much more clear. :grimacing: